### DISCUSSION TITLE
Unknown datatype after conversion from Tensorflow to onnx

### DISCUSSION THREAD

**Discussion by anandvishes** [microsoft/onnxruntime#10583] [Other Q&A] [answered]
Hi,
I tried converting Tensorflow frozen model to ONNX using tf2onnx.convert, the ouput onnx model had a datatype unk_xx.
Is it expected or what should be done to get a valid value. TIA

Command used : python3 -m tf2onnx.convert  --input <model_name> --inputs <input node> --outputs <output_node> --output <output_model>
![input_oonx](https://user-images.githubusercontent.com/63385575/154430111-5f67cfab-3ac3-434e-9b3c-98d2c5b7a75b.JPG)
![op_onnx](https://user-images.githubusercontent.com/63385575/154430249-1154d9ff-3ecd-4fff-8aaf-00bcc8998afe.JPG)

**Comment by snnn** [comment, marked as answer]
unk_xx is variable name, not data type.
